main.py

```diff
@@ -61,6 +64,17 @@ def main():
         if fullscreen:
             tdl.set_fullscreen(not tdl.get_fullscreen())
 
+        # Enemy turn
+        for entity in display_manager.cur_map.entities:
+            entity.take_turn(player, display_manager.cur_map)
+
+        # Check for player death
+        # TODO: Handle player death as a game state
+        if player.hp == 0:
```

Comment:
player/actors should have a `dead` or `alive` attribute

Reply:
Sure.